FlotaLabel: Type definitions are missing a children prop

### Describe the bug

Even though the component clearly accepts children, the props in the accompanying .d.ts file don't contain an entry `children`.

### Reproducer

_No response_

### PrimeReact version

10.6.3

### React version

17.x

### Language

TypeScript

### Build / Runtime

Vite

### Browser(s)

_No response_

### Steps to reproduce the behavior

The example from the docs:

```tsx
<FloatLabel>
    <InputText id="username" value={value} onChange={(e) => setValue(e.target.value)} />
    <label for="username">Username</label>
</FloatLabel>
```

does not compile. The error is

```
error TS2559: Type '{ children: Element; }' has no properties in common with type 'IntrinsicAttributes & FloatLabelProps & RefAttributes<HTMLSpanElement>'
```

### Expected behavior

The code should compile, as it is the intended usage.

--- a/components/lib/floatlabel/FloatLabel.d.ts
+++ b/components/lib/floatlabel/FloatLabel.d.ts
@@ -50,6 +50,11 @@ export interface FloatLabelPassThroughOptions {
  */
 export interface FloatLabelProps {
     /**
+     * Used to get the child elements of the component.
+     * @readonly
+     */
+    children?: React.ReactNode | undefined;
+    /**
      * Used to pass attributes to DOM elements inside the component.
      * @type {FloatLabelPassThroughOptions}
      */

